.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: ruby
+sudo: required
+cache: bundler
+rvm:
+- 2.4.5
+services:
+- docker
+addons:
+  postgresql: '9.5'
```

Comment:
Might want to consider moving to a newer version of PG at some point and also moving away from the MIQ customized image. I don't think we have any need for either this version or  the changes we make to the image...

Reply:
Not for this PR though I think.